docs/reference/google.auth.app_engine.rst

```diff
@@ -1,5 +1,5 @@
-google.auth.app_engine module
-=============================
+google.auth.app\_engine module
+==============================
```

Comment:
That is not valid Sphinx markup.  Please revert.  Note the [current, correctly-rendered page](https://google-auth.readthedocs.io/en/latest/reference/google.auth.app_engine.html).

Reply:
This was added by the tox doc functions (not sure why). Removed.

Reply:
Should I add an issue about this?

Reply:
What is "tox doc"? Ahh, I guess you must have run `tox -e docgen`?

I see that `CONTRIBUTING.rst` suggests running it, but I'm not sure why, since the generated files typically don't contain stuff relevant to "ordinary" fixes like the one in this PR:  it should matter only if new modules are being added.  @theacodes can you comment?

Reply:
Yes, that's correct. I basically ran all the suggested tox commands, including 'tox -e docgen'.  First time I've ever used it, so I've got to admit I don't really understand everything it does.

Reply:
Given that `sphinx-apidoc` produces that markup, I guess I was wrong in my assertion that the `\_` is invalid markup.  But it *is* unnecessary.